README.rst

```diff
@@ -0,0 +1,227 @@
+FunFlow
+=======
+
+.. highlight:: haskell
+.. default-role:: code
+
+.. image:: https://circleci.com/gh/tweag/funflow.svg?style=svg
+    :target: https://circleci.com/gh/tweag/funflow
+
+FunFlow is a library and tools to compose and run computational workflows. 
+A workflow is a computation built up of multiple steps. These steps are then
+wired together into a larger computation. Outputs from previous steps can form
+inputs to subsequent steps. 
+
+Features
+--------
+
+- Funflow is designed to be **integrated** into your application. It allows for
```

Comment:
This feels like it's two separate points.

Reply:
Fixed